src/configuration.ts

```diff
@@ -103,20 +120,27 @@ function getHomeDir() {
     }
 }
 
-export function getRelativePath(directory: string): string {
+export function getRelativePath(directory: string, relativeTo?: string): string {
     if (directory != null) {
-        return path.relative(moduleDirectory, directory);
+        const basePath = relativeTo || process.cwd();
+        return path.resolve(basePath, directory);
     }
 }
 
-export function getRulesDirectories(directories: string | string[]): string[] {
+export function getRulesDirectories(directories: string | string[], relativeTo?: string): string[] {
     let rulesDirectories: string[] = [];
 
     if (directories != null) {
         if (typeof directories === "string") {
-            rulesDirectories = [getRelativePath(<string>directories)];
+            rulesDirectories = [getRelativePath(<string>directories, relativeTo)];
         } else {
-            rulesDirectories = (<string[]>directories).map((dir) => getRelativePath(dir));
+            rulesDirectories = (<string[]>directories).map((dir) => getRelativePath(dir, relativeTo));
+        }
+    }
+
+    for (const directory of rulesDirectories) {
+        if (!fs.existsSync(directory)) {
+            throw new Error(`Could not find custom rule directory: ${directory}`);
```

Comment:
`getRelativePath` is already resolving to absolute paths, so this will show users an absolute path already